packages/frontend/src/common/components/FormTextInput.tsx

```diff
@@ -44,7 +46,9 @@ export function TextInput<T extends FieldValues>({
           className={`input w-full text-sm bg-lightHover text-gray-600 leading-6 placeholder:font-normal focus-within:placeholder:font-normal font-bold ${
             error?.message && '!input-error !bg-error !bg-opacity-20'
           } ${icon && 'pl-12'}`}
-          {...register(name)}
+          {...register(name, {
+            onChange: () => onChange && onChange(),
```

Comment:
Doesn't it do the same thing as before. It seems like it overrides the default onChange 🤔 Then, you can't do the form input validation and call the onChange simultaneously.

Reply:
I found it in documentation and I've tested validation and onChange simultaneously. Works correct.